models/stg_intercom__contact_tag_history.sql

```diff
@@ -32,7 +32,10 @@ final as (
     select 
         contact_id,
         cast(contact_updated_at as {{ dbt.type_timestamp() }}) as contact_updated_at,
-        tag_id
+        tag_id,
+        _fivetran_active,
```

Comment:
I intentionally did not include the filter here because if a customer wants to analyze the historical changes of contact tags then it is still available. We filter these records out in the transform package, but since we didn't filter them out previously in the staging models I wanted to keep that same structure for customers.